package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "operator-workflows",
+  "description": "Github workflows for charm developments",
+  "version": "0.0.0",
+  "author": "",
+  "private": true,
+  "homepage": "https://github.com/canonical/operator-workflows",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/canonical/operator-workflows.git"
+  },
+  "bugs": {
+    "url": "https://github.com/canonical/operator-workflows/issues"
+  },
+  "keywords": [
+    "actions",
+    "node",
+    "setup"
+  ],
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "engines": {
+    "node": ">=20"
+  },
```

Comment:
I think the .node-version file above might not be necessary if i understand correctly? :D

Reply:
I think `.node-version` might still be necessary for tools like `nodenv` or `nvm`. It was copied from https://github.com/actions/typescript-action I think we can keep it here for now.